Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml

```diff
@@ -102,6 +102,39 @@
     seed: towercap
   - type: Log
 
+- type: entity
+  name: nettle
+  description: Stingy little prick.
+  id: Nettle
+  parent: ProduceBase
+  components:
+  - type: Sprite
+    sprite: Objects/Specific/Hydroponics/nettle.rsi
+  - type: Item
+    size: 10
+    sprite: Objects/Specific/Hydroponics/nettle.rsi
+    prefix: inhand
+  - type: MeleeWeapon
+    damage:
+      types:
+        Heat: 10
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        reagents:
+        - ReagentId: Histamine
+          Quantity: 25
+  - type: Produce
+    seed: nettle
+  - type: MeleeChemicalInjector
+    transferAmount: 6 #To OD someone you would need 2 nettles and about 6-7 hits, the DOT is likely to crit them if they are running away with almost no health
+  - type: Extractable
+    grindableSolutionName: "food"
```

Comment:
grindable doesn't match the solution

Reply:
strangely grinding works, but I should just remove the quotes?